src/index.js

```diff
@@ -1,27 +1,35 @@
+/* eslint-disable import/no-cycle */
+
 import './style.css';
+import { addTask, removeTask } from './addRemove.js';
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
- [ ] Linters are often used to maintain code standards. Disabling rules defeats the purpose. Kindly remove these eslint-disables and fix the issues.